blues/app-accelerators
# bug: nf9 fix display wrapping to next line when device UID is in table

# Problem Context

Before a user has renamed their Notecard device from its default Device UID, when it's displayed in the table, the column is just a smidge too short and the ID is forced to line wrap which doesn't look good.

## Changes

Adjust column width so the line wrap no longer occurs.

## Screenshot (if applicable)

![Screenshot 2023-02-28 at 2 36 42 PM](https://user-images.githubusercontent.com/20400845/221960998-29cd4f11-9b09-4e3e-a33f-44aacae5e141.png)

## Testing

Unit / integration / e2e tests? All tests pass

Steps to test manually?

1. Fire up the app locally
2. Change a device name from an assigned name like "Kitchen Sink" to "dev:864622040379949" and save the change.
3. See that the device UID no longer wraps to the second line, but stays on one line on larger viewports or in mobile view (the tablet view still has line wrapping, but that's unavoidable to make the table fit the smaller screen size)


Any other sorts of testing notes to include?

## Any other related PRs

n/a

## Ticket(s)

https://trello.com/c/tDEVFPzd/184-fix-name-input-box-nf9-nf10-so-it-doesnt-wrap-on-device-uid-names
